#!/usr/bin/env python

PACKAGE = "PFT"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("flag_use_gazebo_truth",bool_t,  0, "", True)
gen.add("flag_reset_gazebo",bool_t,  0, "Change to true to reset gazebo models", False)

#gen.add("robot1_name",str_t,  0, "Name Given to First Robot", "agent1")
#gen.add("robot2_name",str_t,  0, "Name Given to Second Robot", "agent2")

gen.add("initial_x1",double_t,   0, "Initial X-coordinate for First Agent",0)
gen.add("initial_x2",double_t,   0, "Initial X-coordinate for Second Agent",0)

gen.add("initial_y1",double_t,   0, "Initial Y-coordinate for First Agent",5)
gen.add("initial_y2",double_t,   0, "Initial Y-coordinate for Second Agent",-5)

exit(gen.generate(PACKAGE, "PFT", "resetParams"))
